test/lib/samson/form_builder_test.rb

```diff
@@ -172,9 +172,9 @@ def fake_erb_rendering
 
   # NOTE: ideally don't use a plugin model, but we need something with accepts_nested_attributes_for
   describe '#fields_for_many' do
-    def render(*args)
+    def render(*args, **kwargs)
       project.rollbar_dashboards_settings.build # TODO: this does not get rendered
-      builder.fields_for_many(*args) do |p|
+      builder.fields_for_many(*args, **kwargs) do |p|
```

Comment:
FYI could use `...` here too

Reply:
> FYI could use `...` here too

Yes. However, I prefer this way as it makes it clear that block arguments are not passed along with args and kwargs.